Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -9,6 +9,13 @@
     state: icon
   - type: Item
     size: 9999
+  - type: Appearance
+  - type: SubFloorHide
+  - type: Anchorable
+  - type: CollideOnAnchor
+    enable: false
```

Comment:
backpacks shouldn't be anchorable

Reply:
None of these extra comps should be here.

Reply:
Removing that would completely negate half the feature of this PR, Making bags stashable. Note: That you can't simply anchor to _any_ tile. It can only be anchored to plating, where you can then place a tile over it. Effectively stashing it.